.github/workflows/ci.yml

```diff
@@ -119,7 +119,9 @@ jobs:
           - '16.5' # last version before unconditional promise fast-path
           - '16.6' # first version after unconditional promise fast-path
           - '18' # Active LTS
-          - '20' # Future LTS
+          # UNTIL https://github.com/nodejs/node/issues/49497 is released (probably 20.6.1)
+          # - '20' # Future LTS
+          - '20.5' # last good version before breakage in 20.6.0
```

Comment:
we know 20.5 is broken in other ways

Reply:
Landed https://github.com/endojs/endo/pull/1786